feature-libs/order/components/order-details/order-detail-reorder/reorder-dialog/reorder-dialog.component.html

```diff
@@ -0,0 +1,81 @@
+<div
+  [cxFocus]="focusConfig"
+  (esc)="close('Escape clicked')"
+  class="cx-modal-container"
+>
+  <div class="cx-modal-content">
+    <ng-container>
+      <div class="cx-dialog-header modal-header">
+        <div class="cx-dialog-title modal-title">
+          {{ 'reorder.dialog.reorderProducts' | cxTranslate }}
+        </div>
+        <button
+          type="button"
+          class="close"
+          attr.aria-label="{{ 'addToCart.closeModal' | cxTranslate }}"
+          (click)="close('Close Import Products Dialog')"
+        >
+          <span aria-hidden="true">
+            <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+          </span>
+        </button>
+      </div>
+    </ng-container>
+
+    <!-- Modal Body -->
+    <div class="cx-dialog-body modal-body">
+      <ng-container *ngIf="!(loading$ | async); else loading">
+        <p>
+          <ng-container *ngIf="!cartModifications.length; else cartResponse">
```

Comment:
There is a discrepency between the observed behaviour and the testing instructions.
`It should open a dialog with the result of items added to cart (success/low stock/not in stock/other)`

There is no feedback of the added items displayed in the modal. The template does not seem to implement this feature either. The expected behaviour needs to be clarified.

Reply:
So the way it currently works is that if we receive a response from the endpoint saying that it could not add all the items to the cart, then we show those messages otherwise we always show the success message "Products have been successfully added to the cart"

Reply:
Spec has been revised to match the implementation.